client/src/components/layout/CustomNavLink.js

```diff
@@ -7,7 +7,7 @@ const CustomNavLink = ({ title, path }) => {
     <>
       <NavLink
         to={`/${path}`}
-        activeClassName={classes.active}
+        activeclassname={classes.active}
```

Comment:
why lowercase?

Reply:
Warning: React does not recognize the `activeClassname` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `activeclassname` instead.